src/silx/gui/plot/items/_arc_roi.py

```diff
@@ -714,6 +727,20 @@ def getGeometry(self):
             raise ValueError("This ROI can't be represented as a section of circle")
         return geometry.center, self.getInnerRadius(), self.getOuterRadius(), geometry.startAngle, geometry.endAngle
 
+    def getPositionRole(self, role: Role) -> Tuple[float, float]:
```

Comment:
`getPositionRole` make thinks it returns a role, what about:
```suggestion
    def getPosition(self, role: Role) -> Tuple[float, float]:
```
or eventually `getPositionFromRole`.